packages/storybook/builder/package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "@modern-js/storybook-builder",
+  "version": "2.31.2",
+  "description": "modern.js support for storybook",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/web-infra-dev/modern.js",
+    "directory": "packages/storybook/builder"
+  },
+  "jsnext:source": "./src/index.ts",
+  "main": "./dist/index.js",
+  "types": "./src/index.ts",
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc --watch --incremental",
+    "test": "vitest run"
+  },
+  "exports": {
+    ".": {
+      "jsnext:source": "./src/index.ts",
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    },
+    "./preset": "./dist/preset.js",
+    "./types": {
+      "jsnext:source": "./src/types.ts",
+      "types": "./dist/types.d.ts",
+      "default": "./dist/types.js"
+    },
+    "./templates/preview.ejs": "./templates/preview.ejs",
+    "./templates/virtualModuleModernEntry.js.handlebars": "./templates/virtualModuleModernEntry.js.handlebars",
+    "./package.json": "./package.json"
+  },
+  "engines": {
+    "node": ">=16.0.0"
```

Comment:
Is Node 14 no longer supported by storybook v7?

Reply:
Yes
![image](https://github.com/web-infra-dev/modern.js/assets/57202839/fbe98db6-4aec-4f31-b66f-9e5042a6b9ab)